src/libgrate/grate-3d.c

```diff
@@ -659,11 +659,13 @@ static void grate_3d_set_texture_desc(struct host1x_pushbuf *pb,
 				      struct host1x_pixelbuffer *pixbuf,
 				      unsigned max_lod,
 				      unsigned wrap_mode,
-				      bool mip_filter,
-				      bool mag_filter,
-				      bool min_filter)
+				      bool mipmap_enabled,
+				      bool min_filter_linear_within,
+				      bool min_filter_linear_between,
+				      bool mag_filter_linear)
```

Comment:
Since I'm not very familiar with OpenGL and related terminology in general, I'm expecting that from you :) Don't shy, always let me know if something feels clumsy.